simfit_data_fullAngularMass_Swave.cc

```diff
@@ -164,19 +164,68 @@ void simfit_data_fullAngularMass_SwaveBin(int q2Bin, int parity, bool multiSampl
   // Random generators
   RooRandom::randomGenerator()->SetSeed(1);
 
+  //
+  // retrieve Z4430 model workspace from file
+  // only a single year sample of Z is available, therefore the same parametrisation is used for the 3 years of data taking
+  //
+  RooAbsPdf* Z4430_ang_pdf = 0;
+  RooArgSet* Z4430_ang_params = 0;
+  RooAbsPdf* Z4430_mass_pdf = 0;
+  RooArgSet* Z4430_mass_params = 0;
+  if (q2Bin==6 && fitOption>0){
+    string filename_Z4430 = "/eos/user/a/aboletti/BdToKstarMuMu/fileIndex/Zmodel/HistZ4430.root";
+    if (!(retrieveWorkspace(filename_Z4430, wsp_Z4430, "wZ4430"))) return;
+
+    // retrieve angular component of the Z pdf
+    Z4430_ang_pdf = (RooAbsPdf*) wsp_Z4430[0]->pdf("Z4430_ang_pdf");
+    if(!Z4430_ang_pdf){
+      std::cout << "Z4430_ang_pdf not found!!!\n" << std::endl;
+      wsp_Z4430[0]->allPdfs();
```

Comment:
I will check this

Reply:
Ok, this can be a ->Print(); indeed. There were a reason for this and I never finish to implement the error condition properly. But, since I don't remember, I agree to substitute with a Print();  
Just for general info: there were different versions of the HistZ4430.root, with different pdf (rt and wt only, angular only, different mass...). This was the reason to introduce a dump of all the pdfs.

Reply:
So maybe now it's not needed any more, and we can remove the line?

Reply:
I would retain with the Print()

Reply:
Here:
```suggestion
      wsp_Z4430[0]->allPdfs().Print();
```